docs/architecture.md

```diff
@@ -141,14 +141,37 @@ With a couple of critical differences:
 
 Execution will proceed similarly as in the Java case.
 
+### submitArmadaSpark.sh Convenience script
+
+There is a convience script ./scripts/submitArmadaSpark.sh, that makes it easier to submit jobs to the cluster.  There are many options; run it with -h to see them all.  By default, it will run the sparkPi example jobs.  To run your own jobs, you can run it like so:
```

Comment:
```suggestion
There is a convience script `./scripts/submitArmadaSpark.sh`, that makes it easier to submit jobs to the cluster.  There are many options; run it with `-h` to see them all.  By default, it will run the sparkPi example jobs.  To run your own jobs, you can run it like so:
```